libs/partners/nvidia-ai-endpoints/pyproject.toml

```diff
@@ -48,6 +48,7 @@ ruff = "^0.1.5"
 mypy = "^0.991"
 langchain-core = {path = "../../core", develop = true}
 types-requests = "^2.31.0.10"
+types-pillow = "^10.2.0.20240125"
```

Comment:
@VKudlay this was all that was needed via `poetry add --group typing types-pillow`!

Mypy actually prints out errors related to this. The langchain-specific bit is the `--group typing` to make sure it only gets added at lint/type checking time instead of getting packages with the whole library.

Reply:
@efriis Fantastic! Thanks so much!